data/Yushi5058.json

```
@@ -5,13 +5,18 @@
   "tags": ["Open Source", "Front-End", "Linux", "Javascript"],
   "socials": [
     { "icon": "FaGithub", "url": "https://github.com/Yushi5058" },
-    { "icon": "SiCodewars", "url": "https://www.codewars.com/users/Yushi5058" }
+    { "icon": "SiProtonmail", "url": "mailto:yushi_61@proton.me" }
   ],
   "links": [
     {
-      "name": "Follow me on Github",
+      "name": "Follow me on my dev journey",
       "url": "https://github.com/Yushi5058",
       "icon": "FaGithub"
+    },
+    {
+      "name": "Let's get in touch",
+      "url": "https://yushi5058.github.io/",
+      "icon": "FaLink"
     }
   ]
-}
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```